stlab/channel.hpp

```diff
@@ -35,9 +35,15 @@ template <typename> class sender;
 template <typename> class receiver;
 
 /**************************************************************************************************/
+/*
+ * close on a process is called when a process is in an await state to signal that no more data is coming. 
+ * In response to a close, a process can switch to a yield state to yield values, otherwise it is destructed.
+ * await_try is await if a value is available, otherwise yield (allowing for an interruptible task).
+ */
 
 enum class process_state {
     await,
+    await_try,
```

Comment:
Why did await_try come back in? away_try is await_immediate which I thought I provided a constexper for... but apparently not.